web/src/integration-test/java/ca/hendriks/bartender/web/functionaltest/ingredient/IngredientSteps.java

```diff
@@ -64,4 +71,9 @@ public void the_ingredients_should_be(final DataTable dataTable) {
         }
     }
 
+    @After
+    public void cleanup(){
+        dsl.ingredients.cleanUpRepository();
+    }
```

Comment:
nice! though I have found putting all the @After code blocks in a singie place, like `DomainSpecificLanguage`, makes them easier to keep track of.